### PR TITLE
Bug-6: Pull-up bar missing

### PR DESCRIPTION
**Pull request by MackTr**
-Pull up bar was missing because of change in architecture, this is the return. Code was adapted to make it a component and to route to the BUsPage and AppSettings page correctly.

### CODE REVIEW

File: SD-K9/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -3,14 +3,6 @@ import { PreloadAllModules, RouterModule, Routes } from '@angular/router';
...
 const routes: Routes = [
   { path: '', redirectTo: 'home', pathMatch: 'full' },
-  {
-    path: 'bus',
-    loadChildren: () => import('src/app/components/bus/bus.module').then(m => m.BusPageModule)
-  },
-  {
-    path: 'settings',
-    loadChildren: () => import('src/app/pages/app-settings/app-settings.module').then(m => m.AppsettingsPageModule)
-  },
```
Comment (paulMoose) [inline SD-K9/src/app/app-routing.module.ts:13]: Why did you remove the routing for bus.page and settings.page?

Comment (MackTr) [reply SD-K9/src/app/app-routing.module.ts:13]: The bus and settings are modal components, they do not need to be accessed by the routing and only by the modal button. This button is on the pull-up bar component. The code was giving me errors when it was there.
